src/cool_seq_tool/sources/uta_database.py

```diff
@@ -61,6 +61,25 @@ class GenomicAlnData(BaseModelForbidExtra):
     alt_strand: Strand = Field(..., description="Strand.")
 
 
+class TxExonAlnData(GenomicAlnData):
+    """Represent data from UTA tx_exon_aln_v view"""
+
+    tx_ac: StrictStr = Field(..., description="Transcript accession.")
+    tx_start_i: StrictInt = Field(
+        ...,
+        description="`tx_ac`'s start index of the exon using inter-residue coordinates.",
+    )
+    tx_end_i: StrictInt = Field(
+        ...,
+        description="`tx_ac`'s end index of the exon using inter-residue coordinates.",
+    )
+    alt_aln_method: StrictStr = Field(
+        ..., description="The alignment method used to compare sequences."
+    )
```

Comment:
are there cases where we aren't using splign? because if it's always splign as a policy matter, I don't think we need to retain this info

Reply:
For Ensembl, we use genebuild